### ISSUE TITLE
Fix weird λ≈0 plot artifacts

### ISSUE THREAD

**Issue by divilian** [WheezePuppet/specstar#67]
When running `param_sweep.jl` with (say) these parameters:
```
param_to_sweep=:λ   
                    
start_value=0.0     
end_value=3         
num_values=30       
trials_per_value=14 
```
some of the plots, including `num_living_agents_pre_num_protos_post_num_protos_pre_num_living_agents_post.png`, look weird on the left-hand side:
![num_living_agents_pre_num_protos_post_num_protos_pre_num_living_agents_post](https://user-images.githubusercontent.com/4355665/60203925-bfbec400-981b-11e9-8a3e-f1095beb1aca.png)

Assuming this is an error, due to some weird artifact when the parameter is near zero, fix the plots. If not, figure out why it is occurring so we can explain it in the manuscript.

**Comment by divilian** [comment]
(@venkatachalapathy also requests that we also check that λ=0 is the same as none-to-none network dynamics. See new issue #68.)

**Comment by divilian** [comment]
Will fixed this in a17bda0.
